integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/status/strategy/DpfStatusStrategy.java

```diff
@@ -53,37 +50,37 @@ public void checkStatus(@NotNull Set<Conversation> conversations) {
     public void updateStatuses(Set<Conversation> conversations) {
```

Comment:
Kanskje den her skulle hatt ein test eller to ?

Reply:
Ja, enig. Skal legge til.